pkg/sysinfo/sysinfo_darwin.go

```diff
@@ -6,8 +6,12 @@ import (
 	"os/exec"
 	"regexp"
 	"strconv"
+
+	"github.com/patrickmn/go-cache"
 )
 
+var sysinfoCache *cache.Cache = cache.New(cache.NoExpiration, cache.NoExpiration)
```

Comment:
If you set this in sysinfo.go then you won't have to define it 3 times.

Reply:
Good point. Will do.